### PR TITLE
Refactored to use helper functions

### PR DESCRIPTION
**Pull request by asnewman**

### CODE REVIEW

**Comment by ktamilvanan** [comment]
FWIW, it looks good to me.

**Comment by asnewman** [comment]
> FWIW, it looks good to me.

Cool, I will merge

File: tests/routesTests/GetChartsTests.py [hunk 1]
```diff
@@ -3,42 +3,25 @@
...
+
+      register_data = register_david_janzen(self.base_url, self.headers)
+      self.assertTrue('id' in register_data)
+      self.assertTrue(register_data['id'] > 0)
 
+      login_data = login_david_janzen(self.base_url, self.headers)
+      self.assertIsNotNone(login_data['userId'])
+      self.assertIsNotNone(login_data['token'])
+
+      self.userId = login_data['userId']
```
Comment (sshaul) [inline tests/routesTests/GetChartsTests.py]: Make sure to change `self.userId` to `self.suser_id` here for consistency
```suggestion
      self.user_id = login_data['userId']
```

**Review by sshaul — APPROVED** [review]
👍

**Pull request merged**